src/LayerEditor/ui/data/regions_model.py

```diff
@@ -42,7 +43,7 @@ def delete_region(self, reg: Region):
             for layer in block.layers:
```

Comment:
Rather should be done out of this method. Not appropriate to the regions_model

Reply:
I would rather leave it here. We discussed it already. I moved it to LEModel and there was some problem with it so I moved it back.